android/anc/src/main/assets/vital_signs_questionnaire_metric.json

```diff
@@ -0,0 +1,500 @@
+{
+  "resourceType": "Questionnaire",
+  "status": "active",
+  "date": "2020-11-18T07:24:47.111Z",
+  "name": "vital-sign",
+  "title": "Vital Sign",
+  "subjectType": [
+    "Patient"
+  ],
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/uv/sdc/StructureDefinition/sdc-questionnaire-targetStructureMap",
+      "valueCanonical": "https://fhir.labs.smartregister.org/StructureMap/383"
+    }
+  ],
+  "item": [
+    {
+      "linkId": "select-vital-sign",
+      "type": "choice",
+      "required": true,
+      "text": "Select vital sign",
+      "extension": [
+        {
+          "url": "http://hl7.org/fhir/StructureDefinition/questionnaire-itemControl",
+          "valueCodeableConcept": {
+            "coding": [
+              {
+                "system": "http://hl7.org/fhir/questionnaire-item-control",
+                "code": "radio-button",
+                "display": "Radio Button"
+              }
+            ],
+            "text": "A control where choices are listed with a button beside them. The button can be toggled to select or de-select a given choice. Selecting one item deselects all others."
+          }
+        },
+        {
+          "url": "http://hl7.org/fhir/uv/sdc/StructureDefinition/sdc-questionnaire-observationExtract",
+          "valueBoolean": true
+        }
+      ],
+      "answerOption": [
+        {
+          "valueCoding": {
+            "code": "weight",
+            "display": "Weight"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "height",
+            "display": "Height"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "blood-oxygen-level",
+            "display": "Blood Oxygen Level"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "blood-pressure",
+            "display": "Blood Pressure"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "pulse-rate",
+            "display": "Pulse Rate"
+          }
+        },
+        {
+          "valueCoding": {
+            "code": "blood-glucose",
+            "display": "Blood Glucose"
+          }
+        }
+      ]
+    },
+    {
+      "linkId": "vital-signs-body-wight_kg",
+      "definition": "http://acme.org/devices/clinical-codes",
+      "text": "Body Weight (kg)",
+      "type": "decimal",
+      "maxLength": 999,
```

Comment:
isn't this the number of characters in the field? Wouldn't want that to be 3 based on this? https://github.com/opensrp/fhircore/issues/528#issuecomment-931581522

Reply:
I have made these changes I got confused between maxLength and maxValue. I have uploaded the updated questionnaire to the local server as well.